integration/onedc_test.go

```diff
@@ -26,7 +26,7 @@ func TestOneDockerCompose(t *testing.T) {
 
 	f.CurlUntil(ctx, "onedc", "localhost:8000", "🍄 One-Up! 🍄")
 
-	f.ReplaceContents("main.go", "One-Up", "Two-Up")
```

Comment:
?

why did this change? this doesn't seem right

Reply:
...oh weird, I think this was a weird merge thing. good catch.